doc: Add local test and doc to CONTRIBUTING guide (#783)
ansys/pyprimemesh: README.md

    model = prime_client.model
```

## Run tests

> To run tests locally, run this command:

Reviewer: ```suggestion
Run tests locally with this command:
```

> ## Run the tests
> 
> To run the test in local you can use the following command:

Reviewer: ```suggestion
## Run tests

To run tests locally, run this command:
```
